react-components/select/step5.md

```diff
@@ -0,0 +1,71 @@
+Another customization option, available for all variants, is custom display and data for the `Select` options.
+
+Begin with customizing the data. 
+
+To change what data is passed around, and back to you on selection, implement a class that contains any data you wish to persist and a `toString` function, which converts the data in the structure to a string format. This allows the component to internally compare options for filtering, and gives some limited customization for the option's display.
+
+1) **Add a `CustomDataState` class.**
+
+Add the class to the `App.js` file, outside of the App class.
+
+<strong>Note:</strong> This class may contain any amount of properties, and these will be passed back as data when an option is selected. The `toString` function tells the select component how to display and compare the data as a string.
+
+<pre class="file" data-target="clipboard">
+class CustomDataState {
+  constructor(name, abbreviation, capital, founded) {
+    this.name = name;
+    this.abbreviation = abbreviation;
+    this.capital = capital;
+    this.founded = founded;
+  }
+  toString = () =>
+    `${this.name} (${this.abbreviation}) - Founded: ${this.founded}`;
+}
+</pre>
+
+<strong>Note:</strong> You may import the `SelectOptionObject` interface from @patternfly/react-core if using TypeScript. This interface would enforce the required `toString` function for `CustomDataState`.
+
+2) **Replace the state's `options` property with the following array that uses `CustomDataState`.**
+
+The state is located in the App class constructor.
+
+<pre class="file" data-target="clipboard">
+options: [
+  &lt;SelectOption
+    value={new CustomDataState("Alabama", "AL", "Montgomery", 1846)}
+  /&gt;,
+  &lt;SelectOption
+    value={new CustomDataState("Florida", "FL", "Tailahassee", 1845)}
+  /&gt;,
+  &lt;SelectOption
+    value={new CustomDataState("New Jersey", "NJ", "Trenton", 1787)}
+  /&gt;
+]
+</pre>
+
+3) **Edit the state's `options` to display something different than the CustomDataState's toString.**
```

Comment:
Confused as to what to do at this step can you make the instructions more clear should I copy the code underneath?